TLM/TLM/UI/IncompatibleModsPanel.cs

```diff
@@ -139,25 +139,16 @@ public void Initialize()
             blurEffect.size = new Vector2(resolution.x, resolution.y);
             blurEffect.absolutePosition = new Vector3(0, 0);
             blurEffect.SendToBack();
-            blurEffect.eventPositionChanged += OnBlurEffectPositionChange;
-            blurEffect.eventZOrderChanged += OnBlurEffectZOrderChange;
-            blurEffect.opacity = 0;
-            blurEffect.isVisible = true;
-            ValueAnimator.Animate("ModalEffect", delegate (float val) { blurEffect.opacity = val; }, new AnimatedFloat(0f, 1f, 0.7f, EasingType.CubicEaseOut));
+            if (blurEffect != null)
```

Comment:
Removing those callbacks will cause that bug with z-order of incompatible mods panel and blurry background when Paradox login notification opens

Reply:
Wait a sec, did i accidentally change the logic?
The intent was to change delegates to lambdas only, and some line breaks

Reply:
As you can see delegates were removed ;)

Reply:
`val => blurEffect.opacity = val` is lambda
`() => blurEffect.Hide()` is another lambda

Reply:
missing
`blurEffect.eventPositionChanged += OnBlurEffectPositionChange;` 
`blurEffect.eventZOrderChanged += OnBlurEffectZOrderChange;`